### ISSUE TITLE
Failure to generate proper semantics tree after scrolling

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#11175]
Sometimes, after scrolling in accessibility mode with two fingers around the homepage of the flutter gallery app on Android, nodes are no longer selectable on the screen. When printing the Semantics Tree in that state, it looks like this:

```
I/flutter ( 1860): SemanticsNode(0; Rect.fromLTRB(0.0, 0.0, 1440.0, 2392.0))
I/flutter ( 1860):  └SemanticsNode(1; Rect.fromLTRB(0.0, 0.0, 411.4, 683.4) scaled by 3.5x)
I/flutter ( 1860):    └SemanticsNode(2; Rect.fromLTRB(0.0, 0.0, 411.4, 683.4))
I/flutter ( 1860):      └SemanticsNode(3 (dirty); Rect.fromLTRB(0.0, 0.0, 411.4, 683.4) (clipped))
```

That's way to short and the dirty node is suspicious.

**Comment by goderbauer** [comment]
This has been fixed.
